themes-default/slim/views/editShow.mako

```diff
@@ -146,6 +146,11 @@ const startVue = () => {
                 const preferred = this.series.config.qualities.preferred.reduce(reducer, 0);
 
                 return allowed | preferred << 16
+            },
+            displayShowUrl() {
+                // @TODO: Change the URL generation to use `this.series`. Currently not possible because
+                // the values are not available at the time of app-link component creation.
+                return window.location.pathname.replace('editShow', 'displayShow') + window.location.search;
```

Comment:
I don't like it, but this is the best way to **avoid** having to add a watch to the `app-link` component (like I did with `quality-chooser`, `file-browser`, `select-list`, etc.)